apps/web/src/pages/index.tsx

```diff
@@ -1,23 +1,73 @@
 import { Layout } from "../helper/layout";
 import type { ReactElement } from "react";
+import Image from "next/image";
+import Link from "next/link";
 
 export default function Landing() {
   return (
-    <>
+    <div className="flex flex-col h-[calc(100vh-6rem)] justify-between content-start gap-9">
       <div>
-        <h1 className="text-9xl font-sora">
-          <span>SK</span>
-          <span className="text-techdev-gradient-radial">TechDev</span>
-        </h1>
-        {/* <Button variant="contained" color="primary">
-          Join us
-        </Button> */}
+        <div>
+          <h1 className="text-[10rem] font-sora -tracking-[11.75px] break-normal">
+            <span>SK </span>
+
+            <span className="text-techdev-gradient-radial font-bold">
+              Tech Dev
+            </span>
```

Comment:
There are no spaces between “SK”, “Tech”, and “Dev” in the design. Instead, there should be word break opportunities (the `wbr` element),